app/boards/arm/bdn9/bdn9_rev2.dts

```diff
@@ -91,15 +110,15 @@
 &flash0 {
 	/*
 	 * For more information, see:
-	 * http://docs.zephyrproject.org/latest/guides/dts/index.html#flash-partitions
+	 * http: //docs.zephyrproject.org/latest/guides/dts/index.html#flash-partitions
```

Comment:
?

Reply:
Yeah, no idea why neovim fubar'd that. Fixed.